### ISSUE TITLE
Build error using primitive 0.7.0.0

### ISSUE THREAD

**Issue by asr** [gregorycollins/hashtables#57]
I'm getting the following error installing `hashtables 1.2.3.2` using using GHC 8.6.5 and `primtive 0.7.0.0`:

```
src/Data/HashTable/Internal/IntArray.hs:23:44: error:
    Module ‘Data.Primitive.Types’ does not export ‘Addr(..)’
   |
23 | import           Data.Primitive.Types     (Addr (..))
   |                                            ^^^^^^^^^
Failed to install hashtables-1.2.3.2
```

Blocking https://github.com/agda/agda/issues/3800.

**Comment by erikd** [comment]
Fixed by  https://github.com/gregorycollins/hashtables/pull/61

**Comment by erikd** [comment]
Fixed in the 1.2.3.4 release.
